agents/ppo.py

```diff
@@ -39,51 +99,64 @@ def create_from_config(cls, config):
             discount_factor=config.get('discount_factor', 0.99),
             ratio_clip=config.get('ratio_clip', 0.2),
             actor_loss_weight=config.get('actor_loss_weight', 1.0),
-            critic_loss_weight=config.get('critic_loss_weight', 1.0)
+            critic_loss_weight=config.get('critic_loss_weight', 1.0),
+            bag_trajectory_memory=BaseBagTrajectoryMemory.create_from_config(config['path_memory']),
+            ppo_input_adapter=BaseInputAdapter.create_from_config(config['input_adapter'])
+        )
+
+    def forward(self, inputs: Dict[str, Any]) -> Tensor:
+        actor_input = self._ppo_input_adapter.get_actor_input(inputs)
+        critic_input = self._ppo_input_adapter.get_critic_input(inputs)
+        next_node, policy = self._actor.forward(**actor_input)
```

Comment:
No, we decided that actor takes not unpacked dictionary but a specific args, am I wrong?